### ISSUE TITLE
Double clicking on "My Session" configuration gives bad offset error

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#854]
On the Console, navigate to `My Session` tab. Select GEM_TEMPOBJ_CACHE_SIZE in right configuration column and double click. Get error below.  Probably should just make double-clicking a no-op. 
```
a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2
[1] OffsetError (AbstractException) >> _signalToDebugger @9 line 6
[2] OffsetError (AbstractException) >> defaultAction @2 line 18
[3] OffsetError (AbstractException) >> _defaultAction @4 line 4
[4] OffsetError (AbstractException) >> _signalWith: @5 line 25
[5] OffsetError (AbstractException) >> signal @2 line 47
[6] String (Object) >> _error:args: @15 line 11
[7] String (Object) >> _errorIndexOutOfRange: @2 line 6
[8] String >> copyFrom:to: @11 line 16
[9] [] in JadeServer >> systemConfigAsDictionary @60 line 25
[10] Array (Collection) >> do: @5 line 10
[11] JadeServer64bit35 (JadeServer) >> systemConfigAsDictionary @14 line 22
[12] JadeServer64bit35 (JadeServer) >> descriptionOfConfigOption: @2 line 4
[13] GsNMethod class >> _gsReturnToC @1 line 11
===============================================
[1] OffsetError (AbstractException) >> _signalToDebugger @9 line 6
	receiver oop(28039169) == a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2

instance variables:
	-actual oop(18) == 2
	-currGsHandler oop(20) == nil
	-gsArgs oop(20) == nil
	-gsDetails oop(20) == nil
	-gsNumber oop(16026) == 2003
	-gsReason oop(28273665) == 'objErrBadOffsetIncomplete'
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-maximum oop(2) == 0
	-messageText oop(28273409) == 'a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2'
	-object oop(28039425) == ''
	-tag oop(20) == nil

temporary variables and parameters:
	blk oop(20) == nil
	res oop(20) == nil
----------------------------------------------------
[2] OffsetError (AbstractException) >> defaultAction @2 line 18
	receiver oop(28039169) == a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2

instance variables:
	-actual oop(18) == 2
	-currGsHandler oop(20) == nil
	-gsArgs oop(20) == nil
	-gsDetails oop(20) == nil
	-gsNumber oop(16026) == 2003
	-gsReason oop(28273665) == 'objErrBadOffsetIncomplete'
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-maximum oop(2) == 0
	-messageText oop(28273409) == 'a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2'
	-object oop(28039425) == ''
	-tag oop(20) == nil

temporary variables and parameters:
----------------------------------------------------
[3] OffsetError (AbstractException) >> _defaultAction @4 line 4
	receiver oop(28039169) == a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2

instance variables:
	-actual oop(18) == 2
	-currGsHandler oop(20) == nil
	-gsArgs oop(20) == nil
	-gsDetails oop(20) == nil
	-gsNumber oop(16026) == 2003
	-gsReason oop(28273665) == 'objErrBadOffsetIncomplete'
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-maximum oop(2) == 0
	-messageText oop(28273409) == 'a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2'
	-object oop(28039425) == ''
	-tag oop(20) == nil

temporary variables and parameters:
	res oop(20) == nil
----------------------------------------------------
[4] OffsetError (AbstractException) >> _signalWith: @5 line 25
	receiver oop(28039169) == a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2

instance variables:
	-actual oop(18) == 2
	-currGsHandler oop(20) == nil
	-gsArgs oop(20) == nil
	-gsDetails oop(20) == nil
	-gsNumber oop(16026) == 2003
	-gsReason oop(28273665) == 'objErrBadOffsetIncomplete'
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-maximum oop(2) == 0
	-messageText oop(28273409) == 'a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2'
	-object oop(28039425) == ''
	-tag oop(20) == nil

temporary variables and parameters:
	inCextensionArg oop(20) == nil
	res oop(20) == nil
----------------------------------------------------
[5] OffsetError (AbstractException) >> signal @2 line 47
	receiver oop(28039169) == a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2

instance variables:
	-actual oop(18) == 2
	-currGsHandler oop(20) == nil
	-gsArgs oop(20) == nil
	-gsDetails oop(20) == nil
	-gsNumber oop(16026) == 2003
	-gsReason oop(28273665) == 'objErrBadOffsetIncomplete'
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-maximum oop(2) == 0
	-messageText oop(28273409) == 'a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:2'
	-object oop(28039425) == ''
	-tag oop(20) == nil

temporary variables and parameters:
----------------------------------------------------
[6] String (Object) >> _error:args: @15 line 11
	receiver oop(28039425) == ''

instance variables:

temporary variables and parameters:
	argList oop(28361473) == anArray( 2)
	cls oop(139777) == OffsetError
	errNum oop(16026) == 2003
	errorSymbol oop(3996161) == #'objErrBadOffsetIncomplete'
	mv oop(10248193) == anArray( OffsetError, 'objErrBadOffsetIncomplete', anArray( 'An indexable object or NSC ', 1, ' was referenced with an ', 'index ', 2, ' that was out of range.'))
	newArgList oop(28361729) == anArray( '', 2)
----------------------------------------------------
[7] String (Object) >> _errorIndexOutOfRange: @2 line 6
	receiver oop(28039425) == ''

instance variables:

temporary variables and parameters:
	anIndex oop(18) == 2
----------------------------------------------------
[8] String >> copyFrom:to: @11 line 16
	receiver oop(28039425) == ''

instance variables:

temporary variables and parameters:
	startIndex oop(10) == 1
	stopIndex oop(18) == 2
----------------------------------------------------
[9] [] in JadeServer >> systemConfigAsDictionary @60 line 25
	receiver oop(28283649) == anExecBlock1

instance variables:

temporary variables and parameters:
	char oop(25884) == $e
	dict oop(28509441) == aDictionary( 'GemDebuggerActive'->'# GemDebuggerActive?#?# Runtime-only config option for the gem, enabled by?#   System gemConfigurationAt:#GemDebuggerActive put: true?# ?# Controls debugging of exce...
	each oop(28509185) == '?'
	i oop(210) == 26
	line oop(28039425) == ''
----------------------------------------------------
[10] Array (Collection) >> do: @5 line 10
	receiver oop(28041729) == anArray( '# CONFIG_WARNINGS_FATAL  ?#  If TRUE then any warning about invalid or out of range entries?#  in a config file is treated as a fatal error terminating the?#  process (gem or stone) that is ...

instance variables:

temporary variables and parameters:
	aBlock oop(28283649) == anExecBlock1
	i oop(186) == 23
----------------------------------------------------
[11] JadeServer64bit35 (JadeServer) >> systemConfigAsDictionary @14 line 22
	receiver oop(37754881) == aJadeServer64bit35

instance variables:
	-classList oop(20) == nil
	-classOrganizers oop(28514561) == anArray( nil, nil, nil, nil)
	-environment oop(2) == 0
	-methodCommandResult oop(20) == nil
	-methodFilters oop(20) == nil
	-methodFilterType oop(20) == nil
	-readStream oop(20) == nil
	-selectedClass oop(20) == nil
	-selections oop(20) == nil
	-writeStream oop(20) == nil

temporary variables and parameters:
	char oop(25884) == $e
	dict oop(28509441) == aDictionary( 'GemDebuggerActive'->'# GemDebuggerActive?#?# Runtime-only config option for the gem, enabled by?#   System gemConfigurationAt:#GemDebuggerActive put: true?# ?# Controls debugging of exce...
	i oop(210) == 26
	line oop(28039425) == ''
	list oop(28041729) == anArray( '# CONFIG_WARNINGS_FATAL  ?#  If TRUE then any warning about invalid or out of range entries?#  in a config file is treated as a fatal error terminating the?#  process (gem or stone) that is ...
	stream oop(28505601) == aGsFile
----------------------------------------------------
[12] JadeServer64bit35 (JadeServer) >> descriptionOfConfigOption: @2 line 4
	receiver oop(37754881) == aJadeServer64bit35

instance variables:
	-classList oop(20) == nil
	-classOrganizers oop(28514561) == anArray( nil, nil, nil, nil)
	-environment oop(2) == 0
	-methodCommandResult oop(20) == nil
	-methodFilters oop(20) == nil
	-methodFilterType oop(20) == nil
	-readStream oop(20) == nil
	-selectedClass oop(20) == nil
	-selections oop(20) == nil
	-writeStream oop(20) == nil

temporary variables and parameters:
	aString oop(28510465) == 'GEM_TEMPOBJ_CACHE_SIZE'
	dict oop(20) == nil
	key oop(20) == nil
	string oop(20) == nil
----------------------------------------------------
[13] GsNMethod class >> _gsReturnToC @1 line 11
	receiver oop(144897) == GsNMethod

instance variables:

temporary variables and parameters:
```

**Comment by LisaAlmarode** [comment]
happens in 3.2.9

**Comment by ericwinger** [comment]
Restored rowan client services code and fixed brittle code to handle extraneous characters in file. Fix will be in Jadeite 3.2.11
